telegram/ext/jobqueue.py

```diff
@@ -56,6 +72,12 @@ def __init__(self, bot, prevent_autostart=None):
         """:type: float"""
         self._running = False
 
+    def timedelta_to_seconds(self, tdelta):
```

Comment:
This method seems to be redundant, especially since this looks like a purely internal method and you already check `isinstance(var, datetime.timedelta)`